simpleranker.py

```diff
@@ -55,16 +59,30 @@ def rank(self, docs: DocumentArray, parameters: Dict, *args, **kwargs):
             for key, group in groups:
                 chunk_match_list = list(group)
                 if self.ranking == 'min':
-                    chunk_match_list = DocumentArray(sorted(chunk_match_list,key=lambda m: m.scores[self.metric]))
+                    chunk_match_list = DocumentArray(
+                        sorted(
+                            chunk_match_list, key=lambda m: m.scores[self.metric].value
+                        )
+                    )
                 elif self.ranking == 'max':
-                    chunk_match_list = DocumentArray(sorted(chunk_match_list,key=lambda m: -m.scores[self.metric]))
+                    chunk_match_list = DocumentArray(
+                        sorted(
+                            chunk_match_list, key=lambda m: -m.scores[self.metric].value
+                        )
+                    )
                 match = chunk_match_list[0]
                 match.id = chunk_match_list[0].parent_id
                 if self.ranking in ['mean_min', 'mean_max']:
-                    scores = [el.scores[self.metric] for el in chunk_match_list]
-                    match.scores[self.metric] = sum(scores) / len(scores)
+                    scores = [el.scores[self.metric].value for el in chunk_match_list]
+                    match.scores[self.metric] = NamedScore(
+                        value=sum(scores) / len(scores), op_name=self.ranking
+                    )
```

Comment:
Is this the intended way in Jina 3? Can we not just assign value directly?
`match.score[self.metric] = sum(scores) / len(scores)`?